.eslintrc.cjs

```diff
@@ -6,8 +6,7 @@ module.exports = {
     ecmaVersion: 2020,
     sourceType: 'module',
   },
-  plugins: ['prettier'],
-  extends: ['eslint:recommended', 'plugin:prettier/recommended'],
+  extends: ['eslint:recommended', 'prettier'],
   rules: {},
```

Comment:
FYI - This is changing from using `eslint-plugin-prettier`'s `recommended` config (which basically runs prettier as an eslint rule), to using [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) which is only used to disable stylistic eslint rules that conflict with prettier formatting.

Reply:
hmm 🤔 could they maybe make the naming here **more** confusing?